go.mod

```diff
@@ -2,7 +2,7 @@ module kubevirt.io/hostpath-provisioner
 
 go 1.23.0
 
-toolchain go1.24.1
+toolchain go1.23.7
```

Comment:
does this not get reverted once you `go mod tidy/vendor`? anyways you should be okay to drop the toolchain and just keep a vague `go 1.23.0`

Reply:
Nope, I did run go mod tidy and go mod vendor to make sure. And it was not reverted.